pkg/runner/local_docker.go

```diff
@@ -107,7 +107,7 @@ func (*LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wr
 		TestInstanceCount:  input.Instances,
 		TestInstanceParams: input.Parameters,
 		TestSidecar:        true,
-		TestArtifacts:      "/artifacts",
+		TestAssetsDir:      "/artifacts",
```

Comment:
Maybe change the mounted dir as well, so that we are consistent?

Reply:
note: "assets" are usually inputs, not outputs. The outputs are usually called "artifacts".

Reply:
I agree artifacts would be better because it's the **output** and not the **input**. We should come up with a decision (see #374). What do you think @raulk?

Reply:
I don’t think there’s an industry norm here. We already use the name “artifact” for build outputs: Docker images and executables, and I think that name is pretty straightforward and clear. Naming these “artifacts” too would be ambiguous. That’s why I proposed “assets”, which can be input assets (e.g. fixtures and configs), or output assets (e.g. log files, traces, dumps, snapshots of network topology, etc.)

Reply:
Basically I don’t want things being consumed or produced from test runs to have two names (artifacts and assets, like proposed). I’d rather use the term _input/output asset_ to refer to a resource handled by a test case.